### DISCUSSION TITLE
Is there a way to 'close all tasks'?

### DISCUSSION THREAD

**Discussion by illettante** [minbrowser/min#2017] [Q&A]
I don't really use tasks, and i end up with heaps of them. Every now and then I remember that tasks is there and the huge list of tasks I imagine sitting in there bugs me so i go in and there's hundreds of tasks in there and i manually close them all individually. Is there a faster way?

**Comment by reimerd** [comment]
Hmmm... Well over a year old and no replies. 🫤 But I have exactly this question. I never use the "tasks" feature, and would like to be able to clear them in one go. Couldn't there be a keyboard shortcut (or something!) that would "delete all tasks"? 

Not a big deal, but would be nice for sure.

**Update**: I see now there is a feature request for this: #2199

**Reply by PalmerAL** [reply]
There was PR for this a while ago, but it got abandoned: #1622

In the meantime, we added a preference ("When Min starts: Open a new blank task") that's supposed to prevent you from ending up with a huge list of old tasks. It doesn't seem to actually be working though, so we should fix that.

**Reply by PalmerAL** [reply]
> In the meantime, we added a preference ("When Min starts: Open a new blank task") that's supposed to prevent you from ending up with a huge list of old tasks. It doesn't seem to actually be working though, so we should fix that.

Fixed in a84277564d21863f2c3acb1f958adba92a7fb6c2.
